app/views/organisations/index.html.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
This is just empty?

Reply:
Yep it's just a placeholder for https://digital.dclg.gov.uk/jira/browse/CLDC-1102 for now

Reply:
Ah, I'll rebase the 1102 in that case